static/js/message_fetch.js

```diff
@@ -25,6 +25,9 @@ function process_result(data, opts) {
         // Even after trying to load more messages, we have no
         // messages to display in this narrow.
         narrow.show_empty_narrow_message();
+    } else {
+        $("#left_bar_compose_reply_button_big").attr('disabled', false);
+        $("#left_bar_compose_reply_button_big").attr('title', 'Reply (r)');
```

Comment:
This should go in `hide_empty_narrow_message` and you should add a call to that function here.